frontend/src/sections/@dashboard/files/FilesListTemplates.js

```diff
@@ -161,20 +181,26 @@ export default function ListTemplates() {
                                 <TableBody>
                                     {filteredRows.map((row) => (
                                         <TableRow
-                                            key={row.Name}
+                                            key={row.name}
                                             sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
                                         >
-                                            <TableCell style={{ width: "75%" }} component="th" scope="row">
-                                                {row.Name}
+                                            <TableCell style={{ width: "55%" }} component="th" scope="row">
+                                                {row.name}
+                                            </TableCell>
+                                            <TableCell style={{ width: "10%" }} component="th" scope="row">
+                                                {capitalize(row.type)}
                                             </TableCell>
-                                            <TableCell style={{ width: "15%" }} align="right">{convertSize(row.Size)}</TableCell>
+                                            <TableCell style={{ width: "10%" }} component="th" scope="row">
+                                                {capitalize(row.format)}
+                                            </TableCell>
+                                            <TableCell style={{ width: "15%" }} align="right">{convertSize(row.size)}</TableCell>
                                             <TableCell style={{ width: "5%" }} align="left" >
-                                                <Button variant="outlined" onClick={() => { handleClickOpen(row.Name) }}>
+                                                {row.type == "template" ? <Button variant="outlined" onClick={() => { handleClickOpen(row.format, row.name) }}>
```

Comment:
Change this line to:
`row.type === "template" && <Button></Button>`
To keep the code style consistent.
Also, `===` is required to avoid warnings.